tutorials/continuous-deployment-pipeline-google-container-engine-with-codeship/index.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Continuous Deployment Pipeline to Google Container Engine using Codeship
+description: Learn how to create a continuous deployment pipeline to Container Engine from Codeship.
+author: kellyjandrews
+tags: CD, Container Engine, Codeship, Pipeline
+date_published: 2017-08-28
+---
+This tutorial explains how to create a Continuous Deployment
+pipeline to Google Container Engine using Codeship. You will
+learn how to deploy a containerized application when new code is merged into the
+master branch and all integration tests have passed.
+
+Continuous Deployment is the practice of automating the delivery of software to
+production after passing a series of tests. Although not for everyone, it should
+be the goal of any software development and DevOps team not restricted by
+regulatory constraints.
+
+## Before you begin
+
+Take the following steps to enable the Google Container Engine API:
+
+1.  Visit the [Container Engine](https://console.cloud.google.com/projectselector/kubernetes)
+    page in the Google Cloud Platform Console.
+1.  Create or select a project.
+1.  Wait for the API and related services to be enabled, which can take several
+    minutes.
+1.  [Enable billing][billing] for your project.
+
+[billing]: https://support.google.com/cloud/answer/6293499#enable-billing
+
+Make sure you have the following:
+
+1.  [Docker CE Version 17.03.01](https://www.docker.com/community-edition)
```

Comment:
This should be a bulleted list rather than an ordered list, since the order isn't significant.